otx/algorithms/action/utils/convert_public_data_to_cvat.py

```diff
@@ -0,0 +1,271 @@
+"""Convert dataset: Public dataset (Jester[RawFrames], AVA) --> Datumaro dataset (CVAT).
+
+It contains lots of hard-coded to make .xml file consumed on Datumaro.
+
+Current Datumaro format for video (CVAT)
+
+root
+|- video_0
+    |- images
+        |- frames_001.png
+        |- frames_002.png
+    |- annotations.xml
+|- video_1
+    |- images
+    |- annotations.xml
+|- video_2
+
+"""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+# pylint: disable=too-many-locals, c-extension-no-member, invalid-name, too-many-statements
+
+import argparse
+import csv
+import os
+import os.path as osp
+import pathlib
+import shutil
+
+import cv2
+from lxml import etree
+
+
+def generate_default_cvat_xml_fields(i, class_idx, video_path, frame_list):
+    """Generate default CVAT xml fields required to make multi-video CVAT format by using Jester, and AVA dataset."""
+    n_frames = len(frame_list)
+    annotations = etree.Element("annotations")
+
+    version = etree.Element("version")
```

Comment:
We can remove the small number of repetitive code lines using a loop because XML needs a hierarchical level. And this code is the temporary code to support the public action dataset. It will be removed if Datumaro supports the action dataset.